### PR TITLE
#163 disable fraud demo

### PR DESCRIPTION
**Pull request by HenkKodde**

### CODE REVIEW

**Comment by rmothilal** [comment]
Henk your tests are not passing

File: src/api/score/handler.js [hunk 1]
```diff
@@ -23,7 +23,9 @@ const randomFraud = () => {
 }
 
 const calculateFraud = (account, account2 = '') => {
-  const fraud = randomFraud()
+  //  const fraud = randomFraud()
+  // #163 disabling Fraud Demo.
+  const fraud = randomFraud(10)
```
Comment (mdebarros) [inline src/api/score/handler.js]: This does not change the functionality. The randomFraud method does not take an input value.

**Comment by HenkKodde** [comment]
Hi - have fixed with Miguel's guidance. All tests have passed.

Regards,


[image: Modusbox_Logos_Modusbox_Vertical_2C.png] <http://www.modusbox.com/>

Henk Kodde

[image: Solution Partner of the Year North America2017.png]
<http://www.modusbox.com/services/>

505 5th Ave S. Suite 300, Seattle, WA 98104

+27(82)780.8062 | henk.kodde@modusbox.com <youremailaddress@modusbox.com>

**Comment by HenkKodde** [comment]
This should take care of the random fraud scoring.

**Comment by HenkKodde** [comment]
Merge changes from feature/#163disableFraudDemo to development branch.
Note hard coded 'High, Low and blacklist still need to be removed from demo. Another story needs to be created for this.

**Pull request merged**